toucan_connectors/trello/trello_connector.py

```diff
@@ -0,0 +1,141 @@
+"""
+Provide a trello connector.
+
+Check https://developers.trello.com/docs/get-started for the official API documentation.
+
+The connector allow you to list in a dataset all the cards of a board with a chosen set of "fields".
+The available fields are listed in the `Field ` object.
+
+The dataset has one line by card and one columns by fields
+"""
+
+import pandas as pd
+from typing import List
+from enum import Enum
+import requests
+import warnings
+
+from toucan_connectors.toucan_connector import ToucanConnector, ToucanDataSource
+
+
+class Fields(str, Enum):
+    name = 'name'
+    url = 'url'
+    lists = 'lists'
+    members = 'members'
+    labels = 'labels'
+
+
+class TrelloDataSource(ToucanDataSource):
+    board_id: str
+    fields_list: List[Fields] = [Fields.name, Fields.url,
+                                 Fields.lists, Fields.members,  Fields.labels]
+    custom_fields: bool = True
+
+
+class TrelloConnector(ToucanConnector):
+    type = "Trello"
+    data_source_model: TrelloDataSource
+
+    key_id: str = ''
+    token: str = ''
+
+    def get(self, path, **customParams):
```

Comment:
should have a more specific name `get_board` maybe ?

Reply:
Yes good idea !